### PR TITLE
feat: logging in github actions

### PR DESCRIPTION
**Pull request by ryan-timothy-albert**

### CODE REVIEW

File: internal/actions/log.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	}
+
+	languages := environment.GetLanguages()
+	languages = strings.ReplaceAll(languages, "\\n", "\n")
+	langs := []string{}
+	if err := yaml.Unmarshal([]byte(languages), &langs); err != nil {
+		fmt.Println("No language provided in github actions config.")
+	}
+	if len(langs) > 0 {
+		request.Tags["language"] = langs[0]
```
Comment (chase-crumbaugh) [inline internal/actions/log.go:67]: Seems like it would be pretty relevant to collect all languages?

**Review by chase-crumbaugh — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -49,6 +49,8 @@ func main() {
 		err = actions.FinalizeDocs()
 	case environment.ActionRelease:
 		err = actions.Release()
+	case environment.ActionLog:
+		err = actions.LogActionResult()
```
Comment (TristanSpeakEasy) [inline main.go:53]: This shouldn't return an error (we don't want to fail anything if we can't send logs) this should be best effort

Comment (ryan-timothy-albert) [reply main.go:53]: The only time it returns an error is if API key is not set. Any GH Action command would already have failed.

Comment (ryan-timothy-albert) [reply main.go:53]: I'm fine removing that though it doesn't really matter

File: internal/actions/log.go (hunk 1 above)
Comment (ryan-timothy-albert) [reply internal/actions/log.go:67]: We can do that in a separate tag. The GH action accepts multiple inputs of languages for generation but only uses one really.

**Pull request merged**